mypy.ini

```diff
@@ -19,8 +19,8 @@ disallow_any_generics = True
 check_untyped_defs = True
 no_implicit_reexport = True
 
-# for strict mypy: (this is the tricky one :-))
-disallow_untyped_defs = True
+# If True, it enforces things like `def __init__(self) -> CLASSNAME` or `def test_() -> None` which does not worth the effort
+disallow_untyped_defs = False
```

Comment:
I really did not mind this. But I guess it's a preference thing.

Reply:
My concern is the amount of time spent fulfilling a condition that has no impact at all since the static analyzers are able to deduce that automatically